src/VCR/LibraryHooks/StreamWrapperHook.php

```diff
@@ -130,6 +128,7 @@ public function stream_read(int $count): string
     public function stream_write(string $data): int
     {
         throw new \BadMethodCallException('No writing possible');
+        return 0;
```

Comment:
That should really be not needed.
Since the method `throws` an exception the line above, the `return 0;` statement is unreachable.

Actually, this might even be detected in a future version of PHPStan.

Out of curiosity, why did you add it?

Reply:
Yeah, that line is not reachable, but method defines `int` return type. Visual Studio Code syntax checker complains about it - and this suppresses the warning. I can remove it, it's not a problem.

Reply:
Yes please. I'm using PHPStorm and it complains because there is some unreachable code :D

Reply:
Done